syne_tune/stopping_criterion.py

```diff
@@ -118,3 +120,89 @@ def __call__(self, status: TuningStatus) -> bool:
                     )
                     return True
         return False
+
+
+class PlateauStopper(object):
```

Comment:
It would be nice to be able to combine this with another stop criterion (stop once any of them return true).

Could be done in another PR.